cypress/e2e/home.cy.ts

```diff
@@ -33,9 +36,37 @@ describe('Test Movapp - Home page', () => {
     checkVisibility('h2', 'Ukrajinská abeceda');
   });
 
-  it('Test of footer of the page', () => {
+  it('Test of languages of the page', () => {
     cy.contains('Movapp mluví dalšími jazyky').should('be.visible');
     cy.contains('Slovensky').should('be.visible');
     cy.contains('Polski').should('be.visible');
   });
-});
+
+  it('Test of the footer of the page', function(){
+    cy.contains('facebook');
+    cy.xpath ("//a[contains(@href, 'facebook')]").click();
+    cy.contains('instagram');
+    cy.xpath ("//a[contains(@href, 'instagram')]").click();
+    cy.contains('twitter');
+    cy.xpath ("//a[contains(@href, 'twitter')]").click();
+    cy.contains('linkedin');
+    cy.xpath ("//a[contains(@href, 'linkedin')]").click();
+    cy.contains('telegram');
+    cy.xpath ("//a[contains(@href, 't.me')]").click();
+
+  
+    cy.xpath ("//a[contains(@href, 'pomahejukrajine')]").click();
```

Comment:
Can be replaced with this and no need for xpath

cy.contains('Pomáhej Ukrajině').should('have.attr', 'href').and('include', 'pomahejukrajine');